web/app/view/DevicesController.js

```diff
@@ -74,28 +74,30 @@ Ext.define('Traccar.view.DevicesController', {
     },
 
     onCommandClick: function () {
-        var device, deviceId, command, dialog, comboStore;
+        var device, deviceId, command, dialog, comboStore, online;
         device = this.getView().getSelectionModel().getSelection()[0];
+        online = device.get('status') === 'online';
         deviceId = device.get('id');
         command = Ext.create('Traccar.model.Command');
         command.set('deviceId', deviceId);
+        command.set('textChannel', !online);
         dialog = Ext.create('Traccar.view.CommandDialog');
         comboStore = dialog.down('form').down('combobox').getStore();
         comboStore.getProxy().setExtraParam('deviceId', deviceId);
```

Comment:
I still think we should add some spaces to the method to make code more readable.